Dockerfile

```diff
@@ -113,4 +110,5 @@ RUN adduser --uid 1000 invenio --gid 0 && \
 USER 1000
 
 # Start the CERN Open Data Portal application
-CMD ["uwsgi --module ${UWSGI_WSGI_MODULE} --socket 0.0.0.0:${UWSGI_PORT} --master --processes ${UWSGI_PROCESSES} --threads ${UWSGI_THREADS} --stats /tmp/stats.socket"]
+# hadolint ignore=DL3025
+CMD uwsgi --module ${UWSGI_WSGI_MODULE} --socket 0.0.0.0:${UWSGI_PORT} --master --processes ${UWSGI_PROCESSES} --threads ${UWSGI_THREADS} --stats /tmp/stats.socket
```

Comment:
Was this change causing any errors? I actually did similar change to some `reana` repos and now wondering if it will break anything

Reply:
Yes, there was a problem. The image was building, but the site was not working. It might be related to expansion of `${...}` variables. I have tried also `CMD ["uwsgi", "--module", ..."` which is the usual way, but to no avail.  So reverted back due to lack of time.